docs/pages/setup/guides/docker.mdx

```diff
@@ -116,12 +117,17 @@ Create Teleport configs and start the process with the following `docker run` co
 # Create local config and data directories for Teleport, which will be mounted
 # into the container.
 $ mkdir -p ~/teleport/config ~/teleport/data
+
+# Download your license file from the Gravitational dashboard and put it in the correct directory.
```

Comment:
Same comment as above: should there be a separate tab for Cloud users here, since these users won't have access to the customer portal?